packages/fetchai/connections/p2p_noise/connection.yaml

```diff
@@ -0,0 +1,27 @@
+name: p2p_noise
+author: fetchai
+version: 0.1.0
+description: The p2p noise connection implements an interface to standalone golang
+  noise node that can exchange aea envelopes with other agents participating in the
+  same p2p network.
+license: Apache-2.0
+aea_version: '>=0.3.0, <0.4.0'
+fingerprint:
+  __init__.py: QmbPzrjd27coFfS2vN9xDL4uARKZWCCmtWvmEuzGKSjxf7
+  aea/api.go: QmYYRrMemRA1x9UsSg2VrofprHyH5oZv9S3FrF6G9jzCC1
```

Comment:
minor detail, but why not `aea_api.go`? More importantly, I assume we expect `aea/api.go` and `noise_node.go` to be reusable by other libraries and applications?

Reply:
it is a workaround to keep only one package in the connection directory to be able to run `go get -d -v .`